src/mlpack/core/data/load_arff_impl.hpp

```diff
@@ -103,7 +105,7 @@ void LoadARFF(const std::string& filename,
           // `origDimType` string here instead (which has not had ::tolower used
           // on it).
           types.push_back(true);
-          boost::trim_if(origDimType,
+	  boost::trim_if(origDimType,
               [](char c)
               {
                   return c == '{' || c == '}' || c == ' ' || c == '\t';
```

Comment:
We should not forget to handle this at the end of the project. I am just adding comment here so we do not forget about it :+1: